lib/MailQueueHandler.php

```diff
@@ -435,6 +435,10 @@ protected function parseEvent($lang, IEvent $event) {
 			try {
 				$event = $provider->parse($lang, $event);
 			} catch (\Throwable $e) {
+				if ($e instanceof \InvalidArgumentException) {
+					// \InvalidArgumentException is part of the activity API, so we let it through
+					throw $e;
+				}
```

Comment:
```suggestion
			} catch (\InvalidArgumentException $e) {
				/* Ignore */
			} catch (\Throwable $e) {
```
Like this it would restore previous behavior of ignoring those errors silently.
If you throw them where are they catched?

Reply:
the method is documented to throw InvalidArgumentException, but I agree it's cleaner to not rethrow them.